lib-extra/src/main/java/com/diffplug/spotless/extra/java/EclipseFormatterStep.java

```diff
@@ -47,14 +40,14 @@ private EclipseFormatterStep() {}
 	private static final String FORMATTER_METHOD = "format";
 
 	/** Creates a formatter step for the given version and settings file. */
-	public static FormatterStep create(File settingsFile, Provisioner provisioner) {
-		return create(defaultVersion(), settingsFile, provisioner);
+	public static FormatterStep create(Iterable<File> settingsFiles, Provisioner provisioner) {
```

Comment:
This breaks binary compat, but it's very easy to have both methods.

```java
public static FormatterStep create(File settingsFile, Provisioner provisioner) {
    return create(Collections.singleton(settingsFile), provisioner)
}
```

This way we keep binary compat between releases, and we don't have to modify as many tests.